types/braintree-web/index.d.ts

```diff
@@ -1165,8 +1166,8 @@ declare namespace braintree {
      */
     create(options: { client: Client }): Promise<ThreeDSecure>;
     create(options: { client: Client }, callback: callback): void;
-    create(options: { version: number, client: Client }): Promise<ThreeDSecure>;
-    create(options: { version: number, client: Client }, callback: callback): void;
+    create(options: { version?: 1 | '1' | 2 | '2' | '2-bootstrap3-modal' | '2-inline-iframe', client?: Client }): Promise<ThreeDSecure>;
```

Comment:
still missing authorization param here

Reply:
added with update, sorry to keep you busy :)